src/plugins/util/web.js

```diff
@@ -41,6 +44,9 @@ const web = {
     this.patch(req)
 
     const span = startSpan(tracer, config, req, res, name)
+    if (!config.filter(req.url)) {
+      span.context()._sampling.pleaseDrop = true
```

Comment:
Just `drop`. No need to be polite towards a machine (yet).

Reply:
😁